### ISSUE TITLE
CPO训练的eval loss对不齐

### ISSUE THREAD

**Issue by zhongwenzrex-cmd** [modelscope/ms-swift#9241]
### Checklist / 检查清单

- [x] I have searched existing issues, and this is a new bug report. / 我已经搜索过现有的 issues，确认这是一个新的 bug report。

### Bug Description / Bug 描述

在使用beta为0.4，NLL的系数alpha为0.1的情况下，eval NLL loss在上升，eval reward margin在下降，但是eval loss却是在下降，难道不应该也是上升吗？

### How to Reproduce / 如何复现

4.0.3版本上观察到的

**Comment by Jintao-Huang** [comment]
发一下shell

**Comment by zhongwenzrex-cmd** [comment]
PYTORCH_CUDA_ALLOC_CONF='expandable_segments:True' \
    CUDA_VISIBLE_DEVICES=0,1,2,3 \
    NPROC_PER_NODE=$nproc_per_node \
    swift rlhf \
        --rlhf_type cpo \
        --model "$MODEL" \
        --output_dir "$OUTPUT_DIR" \
        --dataset "${DATASETS[@]}" \
        --train_type lora \
        --target_modules all-linear \
        --lora_rank 8 \
        --lora_alpha 32 \
        --torch_dtype bfloat16 \
        --deepspeed zero3 \
        --num_train_epochs 4 \
        --per_device_train_batch_size 1 \
        --per_device_eval_batch_size 1 \
        --learning_rate 1e-5 \
        --freeze_vit false \
        --freeze_aligner false \
        --gradient_accumulation_steps $((32 / nproc_per_node)) \
        --eval_steps 10 \
        --save_steps 20 \
        --save_total_limit 20 \
        --save_only_model false \
        --attn_impl flash_attn \
        --logging_steps 1 \
        --max_length 8192 \
        --warmup_ratio 0.03 \
        --dataloader_num_workers 8 \
        --split_dataset_ratio 0.1 \
        --beta 0.4 \
        --cpo_alpha 0.1


我微调的是经过swift SFT后的Qwen3-VL-2B-Instruct模型

**Comment by zhongwenzrex-cmd** [comment]
<img width="671" height="451" alt="Image" src="https://github.com/user-attachments/assets/9b09785b-0150-4553-a64c-3e143e2f09bd" />，

<img width="680" height="451" alt="Image" src="https://github.com/user-attachments/assets/75ae93ce-0734-4515-bb12-8deb6371be9c" />，

<img width="680" height="451" alt="Image" src="https://github.com/user-attachments/assets/8f111294-bc52-4f8c-9309-d842b655c719" />，没有运行完我就卡断了

**Comment by Jintao-Huang** [comment]
learning_rate use 1e-4

**Comment by zhongwenzrex-cmd** [comment]
Thanks, but this does not explain why such wierd phenonmenon happened or it looks normal to you?
